airbyte-ci/connectors/pipelines/tests/test_helpers/test_utils.py

```diff
@@ -183,8 +183,7 @@ async def test_check_path_in_workdir(dagger_client):
         .with_workdir(str(connector.code_directory))
     )
     assert await utils.check_path_in_workdir(container, "metadata.yaml")
-    assert await utils.check_path_in_workdir(container, "pyproject.toml")
-    assert await utils.check_path_in_workdir(container, "poetry.lock")
+    assert await utils.check_path_in_workdir(container, "manifest.yaml")
```

Comment:
Updated this test as this connector is now manifest-only and does not have poetry files